### PR TITLE
Implement Object Tracker node

### PR DESCRIPTION
**Pull request by DmitrySamoylov**
Story details: https://app.clubhouse.io/lumeo/story/1895

### CODE REVIEW

File: pipeline/src/node_properties/track_properties.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "tracker", rename_all = "snake_case")]
+pub enum TrackProperties {
```
Comment (zeenix) [inline pipeline/src/node_properties/track_properties.rs:5]: it'd be more future safe if we use a struct and have `tracker` as a field. Unless we're very sure this is the only data that will be needed here?

**Review by zeenix — CHANGES_REQUESTED** [review]

File: pipeline/src/node_properties/track_properties.rs (hunk 1 above)
Comment (devarshi) [reply pipeline/src/node_properties/track_properties.rs:5]: likely will have more data in the future

File: pipeline/src/node_properties/track_properties.rs [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    NvDcf,
```
Comment (zeenix) [inline pipeline/src/node_properties/track_properties.rs:6]: @devarshi talking of future-safety, we probably should use generic names here, so NvDCF -> DCF. The other two are generic already.

Comment (DmitrySamoylov) [reply pipeline/src/node_properties/track_properties.rs:6]: Yeah, good catch.

Comment (devarshi) [reply pipeline/src/node_properties/track_properties.rs:6]: Yes, good point.

File: pipeline/src/node_properties/track_properties.rs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    /// Discriminative Correlation Filter (DCF) tracker uses a correlation filter-based
+    /// online discriminative learning algorithm, coupled with a Hungarian algorithm for
+    /// data association in multi-object tracking.
+    Dcf,
+
+    /// Kanade Lucas Tomasi (KLT) algorithm
+    Klt,
+
+    /// The Intersection of Union (IOU) tracker uses the intersection of the detector’s
+    /// bounding boxes across frames to determine the object’s unique ID.
```
Comment (zeenix) [inline pipeline/src/node_properties/track_properties.rs:20]: Added docs too, awesome!

**Review by zeenix — APPROVED** [review]

**Review by devarshi — APPROVED** [review]

**Pull request merged**